test/api/OperatorApi/orderStatus.test.ts

```diff
@@ -0,0 +1,208 @@
+import { RawOrder } from 'api/operator'
+import { getOrderStatus } from 'api/operator/utils'
+
+import { mockTimes, DATE } from '../../testHelpers'
+
+// TODO: add to test data and use it everywhere
+const BASE_ORDER: RawOrder = {
+  creationDate: '2021-01-20T23:15:07.892538607Z',
+  owner: '0x5b0abe214ab7875562adee331deff0fe1912fe42',
+  uid: 'asdasdasd',
+  buyAmount: '0',
+  executedBuyAmount: '0',
+  sellAmount: '0',
+  executedSellAmount: '0',
+  feeAmount: '0',
+  executedFeeAmount: '0',
+  invalidated: false,
+  sellToken: '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984',
+  buyToken: '0xc7ad46e0b8a400bb3c915120d284aafba8fc4735',
+  validTo: 0,
+  appData: 0,
+  kind: 'sell',
+  partiallyFillable: false,
+  signature:
+    '0x04dca25f59e9ac744c4093530a38f1719c4e0b1ce8e4b68c8018b6b05fd4a6944e1dcf2a009df2d5932f7c034b4a24da0999f9309dd5108d51d54236b605ed991c',
+}
+
+describe('Filled status', () => {
```

Comment:
I think you have missing the case where if the order is expired and filled, it's "Filled".

Reply:
All orders are expired in this unittest, so I was missing the other way around (filled and NOT expired).
Added.